Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -16,19 +23,22 @@ def csv_file_to_indicator_list(file_path, col_num, starting_row, auto_detect, de
     with open(file_path) as csv_file:
         file_reader = csv.reader(csv_file)
         for row in file_reader:
-            if line_index >= starting_row:
+            if line_index >= starting_row + offset and len(row) != 0:
```

Comment:
is it possible to slice the `file_reader` object to start from `starting_row + offset`?

Reply:
not that I could find in docs nor examples- https://docs.python.org/3/library/csv.html